### PR TITLE
Adds craftable glass floors for both plasma glass and reinforced plasma glass

### PR DESCRIPTION
**Pull request by Paxilmaniac**
## About The Pull Request
So there I was mapping when I had a thought, "Wow this area would look better if I had plasma glass floors"

And thus, an hour of my life vanished coloring glass purple:
image

pglass

prglass

As an aside, since this is a for my own sake glass floor pr, icemoon and airless types have been added to all windows

## Why It's Good For The Game
Purple glass floors can be cool, especially if you want to use them alongside normal plasma windows.

Plus, the little mapping qol for icemoon and airless types to glass floors, which won't be in the changelog as its not a player facing change.

## Changelog

:cl:
add: Glass floor tiles can now be made out of both plasma glass, and reinforced plasma glass
/:cl:

### CODE REVIEW

File: code/game/turfs/open/floor/glass.dm [hunk 1]
```diff
@@ -39,14 +45,44 @@
...
-	icon = 'icons/turf/floors/reinf_glass.dmi'
-	icon_state = "reinf_glass-0"
-	base_icon_state = "reinf_glass"
-	floor_tile = /obj/item/stack/tile/rglass
 	initial_gas_mix = "ICEMOON_ATMOS"
 
-/turf/open/floor/glass/reinforced/setup_broken_states()
-	return list("reinf_glass-damaged1", "reinf_glass-damaged2", "reinf_glass-damaged3")
+/turf/open/floor/glass/reinforced/airless
+	initial_gas_mix = "AIRLESS_ATMOS"
```
Comment (san7890) [inline code/game/turfs/open/floor/glass.dm:55]:
```suggestion
	initial_gas_mix = "AIRLESS_ATMOS"
	
```

Please add a new line here

**Comment by private-tristan** [comment]
do they show pipes underneath them because showing pipes underneath looks messy

**Comment by VladinXXV** [comment]
where is titaniumglass and plastitaniumglass, glass boy? GET COLORING.

**Comment by Paxilmaniac** [comment]
> where is titaniumglass and plastitaniumglass, glass boy? GET COLORING.

no :heart:

**Review by Twaticus — APPROVED** [review]

**Pull request merged**